app/controllers/navigation/Navigator.scala

```diff
@@ -273,6 +333,28 @@ object Navigator {
         common.orElse(specific)(page)(mode)
     }
 
+  def backLink(page: DeclarationPage, mode: Mode, cacheModel: ExportsDeclaration)(implicit request: JourneyRequest[_]): Call =
```

Comment:
As I mentioned yesterday `cacheModel` is in the request. You're taking this from the same request to pass to the method.

Reply:
WIthout this change, you don't need to add this cache model as argument anywhere